client/src/components/DashboardCoponent/Header.jsx

```diff
@@ -28,28 +30,74 @@ const useStyles = makeStyles({
     width: '180px',
     height: '30px',
   },
+  menu: {
+    marginTop: '35px',
+  },
+  menuButton: {
+    marginRight: '10px',
+  },
 });
 
 function Header() {
   const classes = useStyles();
   const { loggedInUser } = useAuth();
+  const history = useHistory();
+  const popupState = usePopupState({ variant: 'popover', popupId: 'Menu' });
+  const { logout } = useAuth();
```

Comment:
You can import logout using the hook above (line 43) :)

Reply:
Oh true, good point!